starter-commands/deschtimes.js

```diff
@@ -0,0 +1,55 @@
+// index.js
+import { Application } from 'interaction-kit';
+
+export default new Application({
+  applicationID: process.env.APPLICATION_ID,
+  publicKey: process.env.PUBLIC_KEY,
+  token: process.env.TOKEN,
+})
+  .loadDirectory('/commands')
+  // .startServer() // localhost:3000
+  // custom server, passed a fastify app (for vercel:)
+  .startServer(app => {
+    return async (req, res) => {
+      await app.ready();
+      app.server.emit('request', req, res);
+    };
+  });
+
+// commands/cmd.js
+import { Command, Option, OptionType, Embed } from 'interaction-kit';
+
+export default new Command({
+  name: 'deschtimes',
+  description: 'Interact with the Deschtimes tracking system',
+})
+  .subcommand({
+    name: 'list',
+    description: 'Get a list of all active shows',
+    handler: async interaction => {
+      const embed = new Embed({
+        // ...
```

Comment:
I don't know if it's a good idea to provide non-functional commands as starter examples.

Reply:
Yeah, this one will get removed in a follow-up PR - mostly here as a guide for something I want to try